src/deploy-tes-on-azure/Deployer.cs

```diff
@@ -15,6 +15,7 @@
 using Azure.Security.KeyVault.Secrets;
 using Azure.Storage;
 using Azure.Storage.Blobs;
+using deploytesonazure;
```

Comment:
this can be removed once the namespace is changed to TesDeployer

Reply:
Already removed.